charts/accurate/templates/clusterrole.yaml

```diff
@@ -0,0 +1,96 @@
+apiVersion: rbac.authorization.k8s.io/v1
```

Comment:
This ClusterRole should be generated with controller-gen actually.
Could you auto-generate the template from the output of controller-gen?